### ISSUE TITLE
Docs

### ISSUE THREAD

**Issue by timholy** [JuliaImages/ImageDraw.jl#16]
Not required before the end of GSoC, but ideally we'd get this package documented on the JuliaImages site.

**Comment by annimesh2809** [comment]
Yes, I will do that once we complete and register `ImageSegmentation`.

**Comment by tosalonijain** [comment]
@timholy I would like to work on it. Shall I?
